### DISCUSSION TITLE
Run/Debug not running from test explorer

### DISCUSSION THREAD

**Discussion by glend1** [orta/vscode-jest#799] [Q&A] [answered]
I'm sure I have some kind of configuration error, and I wouldn't have know this was possible if I hadn't noticed some screenshots in the manual. I also don't have the ability to run a test by right clicking the test in the margin. 

In the test explorer I just have the option to debug, which fails for an unknown reason.

How do I enable this feature?

**Comment by connectdotz** [comment, marked as answer]
Sounds like you are in the default watch mode, which will run tests automatically with the watchman, thus the manual run menu is disabled. See [How to trigger the test run](https://github.com/jest-community/vscode-jest#how-to-trigger-the-test-run), specifically the [autoRun](https://github.com/jest-community/vscode-jest#autorun) settings, to change it if you need more control. 

> In the test explorer I just have the option to debug, which fails for an unknown reason.

You should be able to debug any test in the explorer... Right now the debug feature works for debugging at the test/describe level, not file nor folder. If you still have problems with the debugging, feel free to file an issue.
